x-pack/plugins/ml/public/explorer/explorer.html

```diff
@@ -115,22 +154,34 @@ <h4 class="euiTitle euiTitle--small">No {{swimlaneViewByFieldName}} influencers
           <br /><br />
         </div>
   
-        <span class="panel-title euiText">
-          Anomalies
-        </span>
+        <span
+          class="panel-title euiText"
+          18n-id="xpack.ml.explorer.anomaliesTitle"
+          i18n-default-message="Anomalies"
+        ></span>
```

Comment:
This doesn't get rendered correctly, the string "Anomalies" doesn't end up as inner HTML/text of the span element.

![image](https://user-images.githubusercontent.com/230104/50635034-e1b44980-0f50-11e9-83c0-7f970245b327.png)

![image](https://user-images.githubusercontent.com/230104/50635039-e842c100-0f50-11e9-8be9-9a77c48d1e93.png)